src/Model/Table/RequestsTable.php

```diff
@@ -79,22 +80,23 @@ public function gc()
         if (time() % 100 !== 0) {
             return;
         }
-        $purge = $this->find()
+        $noPurge = $this->find()
             ->select(['id'])
             ->hydrate(false)
-            ->where(['requested_at <=' => new \DateTime('-2 weeks')])
+            ->order(['requested_at' => 'desc'])
+            ->limit(Configure::read('DebugKit.requestCount') ?: 20)
```

Comment:
True, but then it is even pointless to keep more than 10, as it stands.

On the other hand, is there a way to change how many requests are shown?

Or how about a "Show all" button to show all kept entries in the request history?
This way it makes sense again to keep more than it ormally displays...